src/main/java/frc/robot/RobotManager.java

```diff
@@ -24,6 +25,8 @@
  */
 public class RobotManager extends LoggedRobot {
 
+	public static int roborioCycles;
```

Comment:
You should probably initialize this value at 0, currently it will through a runtime error

Reply:
it works, im pretty sure 0 its just the default value

Reply:
you are correct, `roborioCycles` will be initialized with `int`'s default value(`0`). However I do think you should be explicitly setting `roborioCycles` to `0`, for improved code clarity

Reply:
^
 |

Reply:
I don't think it's necessary I think it's pretty clear that way and that would just be like x=x no point in that